libr/util/sys.c

```diff
@@ -1143,7 +1143,21 @@ R_API bool r_sys_tts(const char *txt, bool bg) {
 R_API const char *r_sys_prefix(const char *pfx) {
```

Comment:
we should probably return char * in the heap here. this static char *thing will fail if we use more than one rsysprefix in the same function or so